pom.xml

```diff
@@ -93,12 +93,6 @@
 		</dependency>
 
 		<dependency>
-		    <groupId>com.google.guava</groupId>
```

Comment:
could you add some lines in the description about what happens here, regarding the quava dependency and what is prevented by removing it?